indigo_app/views/tasks.py

```diff
@@ -100,9 +100,29 @@ def get_context_data(self, **kwargs):
         # merge actions and comments
         actions = task.action_object_actions.all()
         task_content_type = ContentType.objects.get_for_model(self.model)
-        comments = Comment.objects\
+        comments = list(Comment.objects\
             .filter(content_type=task_content_type, object_pk=task.id)\
-            .select_related('user')
+            .select_related('user'))
+        
+        # get the annotation for the particular task
+        task_annotation = Annotation.objects.filter(task=task).first()
```

Comment:
This is a 1-1 relationship (https://github.com/laws-africa/indigo/blob/master/indigo_api/models.py#L1078) so you can simply use `task.annotation`, but you do need to guard against that throwing an `Annotation.DoesNotExist` exception.

Reply:
Yes, I have to guard against that, I had initially done an `Annotation.objects.get()` but I realized I have to guard against that exception too